src/sc_kpm/utils/action_utils.py

```diff
@@ -32,11 +38,26 @@ def check_action_class(action_class: Union[ScAddr, Idtf], action_node: ScAddr) -
     return len(search_results) > 0
 
 
-def get_action_answer(action: ScAddr) -> ScAddr:
-    templ = ScTemplate()
+def get_arguments(action_node: ScAddr, count: int) -> List[ScAddr]:
+    keynodes = ScKeynodes()
+    arguments = []
+    for index in range(1, count + 1):
+        rrel_i = keynodes[f"rrel_{index}"]
+        argument = get_element_by_role_relation(action_node, rrel_i)
+        arguments.append(argument)
+    return arguments
+
+
+def create_answer(action_node: ScAddr, *elements: ScAddr) -> None:
```

Comment:
same, I think all action utils should have an _action_ part in the function name

Reply:
Done